bindings/cs/MongoDB.Libmongocrypt/MongoDB.Libmongocrypt.csproj

```diff
@@ -73,13 +73,6 @@
     </Content>
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
it looks like the current solution requires nuget files in the global nuget storage to run local tests which is not correct. The local tests should be independent in this sense (as it was before). 
It's similar to say that to run local driver tests we need to add nuget files into global nuget storage firstly and only then be able to run unit tests. Please check it.

Reply:
As we tested offline, tests are working fine (in test projects .dll is copied into output). But your comment made me recheck all the .dll paths and I figured out that by removing .targets I broke non SDK project restore, which I am fixing in the second commit.